src/styles/_catalog.scss

```diff
@@ -14,9 +14,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
```

Comment:
You should specify changeable properties
For example: transition: height 300ms, transform 300ms;

Reply:
added